### ISSUE TITLE
Prediction: memory leak of ObstacleClusters member lane_obstacles_

### ISSUE THREAD

**Issue by johnnyleeRH** [ApolloAuto/apollo#13290]
**Describe the bug**
Apollo 6.0, there is no way to clear or limit the ObstacleClusters::lane_obstacles_ increasement

**To Reproduce**
Just Run Predcition module

**Expected behavior**
lane_obstacles_  memory is limited

**Desktop (please complete the following information):**
 - OS: ubuntu

**Comment by daohu527** [comment]
It seems that there is no clear operation for `lane_obstacles_`?

https://github.com/ApolloAuto/apollo/blob/a29a563e95944b603ab9be338cce46e6486a89be/modules/prediction/container/obstacles/obstacle_clusters.h#L126-L131

**Comment by johnnyleeRH** [comment]
> It seems that there is no clear operation for `lane_obstacles_`?
> 
> https://github.com/ApolloAuto/apollo/blob/a29a563e95944b603ab9be338cce46e6486a89be/modules/prediction/container/obstacles/obstacle_clusters.h#L126-L131

Yes, it's true, there is a Init function declaration,  but no implementation

**Comment by storypku** [comment]
@kechxu , would you please help with this issue ?
